docs/source/general/freeradius.rst

```diff
@@ -99,6 +99,14 @@ Configure the ``authorize`` and ``authenticate`` section in the default site
     # this section can be left empty
     authenticate {}
 
+    accounting {
+		uri = "${..connect_uri}/api/accounting/"
```

Comment:
ensure the spaces/tabs are consistent with the other examples, it looks like these lines have too many spaces (other examples have 4).